azure-pipelines.yml

```
@@ -268,10 +268,20 @@ jobs:
     condition: eq(variables.nightlyRelease, true)
     displayName: 'Change name to giotto-tda-nightly'
 
+  # Use the boost_1_72_0-msvc-14.1-64.exe for Windows 2016
+  # Following issue https://github.com/actions/virtual-environments/issues/2667
+  # Now it necessary to download manually Boost, great work ! ...
```

```suggestion
  # As of March 2021, it is necessary to download Boost manually
```
The sarcasm is funny now but may make less sense in the future :)